postfix/files/mapping.j2

```diff
@@ -21,6 +21,7 @@
 {%- else %}
 {#- Some settings need order, handle OrderedDict #}
 {% for item in data %}
```

Comment:
You can simply unpack data in the `for` loop right away:
```jinja
{% for k, v in data %}
{{ format_value(k, v) }}
{%- endfor -%}
```

Reply:
> You can simply unpack data in the `for` loop right away:

Just tried that. It won't fly in Python 3.6:

```
ValueError: not enough values to unpack (expected 2, got 1)
```

Reply:
Yeah, I have mixed the list of dicts with the list of tuples. Your changes definitely looks great.